<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QTextEdit" name="MainTextBox"/>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="buttonDock">
      <item>
       <widget class="QPushButton" name="NewButtonButton">
        <property name="text">
         <string>NewButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="BottomDock">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">Margin: 0px;</string>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="bottomLayout">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPlainTextEdit" name="MainCommandBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="RightDock">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="rightLayout">
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="LeftDock">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="leftLayout">
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>826</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Settings"/>
    <addaction name="actionLoad_Settings"/>
   </widget>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>New</string>
    </property>
    <addaction name="actionButton"/>
    <addaction name="actionCommand_Window"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionShortcuts"/>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="actionBoL"/>
    <addaction name="actionEoL"/>
    <addaction name="actionDL"/>
    <addaction name="actionBoF"/>
    <addaction name="actionEoF"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuNew"/>
   <addaction name="menuEditor"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As ...</string>
   </property>
  </action>
  <action name="actionButton">
   <property name="text">
    <string>Button</string>
   </property>
  </action>
  <action name="actionCommand_Window">
   <property name="text">
    <string>Command Window</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionBeginning_of_Line">
   <property name="text">
    <string>Beginning of Line</string>
   </property>
  </action>
  <action name="actionEnd_of_Line">
   <property name="text">
    <string>End of Line</string>
   </property>
  </action>
  <action name="actionDelete_Line">
   <property name="text">
    <string>Delete Line</string>
   </property>
  </action>
  <action name="actionBeginning_of_File">
   <property name="text">
    <string>Beginning of File</string>
   </property>
  </action>
  <action name="actionEnd_of_File">
   <property name="text">
    <string>End of File</string>
   </property>
  </action>
  <action name="actionBeginning_Of_Line">
   <property name="text">
    <string>Beginning Of Line</string>
   </property>
  </action>
  <action name="actionBoL">
   <property name="text">
    <string>Beginning of Line</string>
   </property>
  </action>
  <action name="actionEoL">
   <property name="text">
    <string>End of Line</string>
   </property>
  </action>
  <action name="actionDL">
   <property name="text">
    <string>Delete Line</string>
   </property>
  </action>
  <action name="actionBoF">
   <property name="text">
    <string>Beginning of File</string>
   </property>
  </action>
  <action name="actionEoF">
   <property name="text">
    <string>End of File</string>
   </property>
  </action>
  <action name="actionSave_Settings">
   <property name="text">
    <string>Save Settings</string>
   </property>
  </action>
  <action name="actionLoad_Settings">
   <property name="text">
    <string>Load Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
